src/ios/GeolocationPlugin.m

```diff
@@ -0,0 +1,314 @@
+        
+        //NSString *insertSQL = [NSString stringWithFormat:@"INSERT INTO GPSTable values(null, '%.6f', '%.6f', '%.6f', '%@', '%@')", 15.12598, 524.12368, 15.6665, @"20160208T26:15:15", @"YES"];
+        
+        const char *insert_stmt = [insertSQL UTF8String];
+        
+        sqlite3_prepare_v2(contactDB, insert_stmt, -1, &statement, NULL);
+        if (sqlite3_step(statement) == SQLITE_DONE)
+        {
+            NSLog(@"Success to add contact");
+        } else {
+            NSLog(@"Failed to add contact");
+        }
+        sqlite3_finalize(statement);
+        sqlite3_close(contactDB);
+    }
+}
+-(void)MakeJSON_Make{
+    JSONArray = [[NSMutableArray alloc] init];
+    if (sqlite3_open([dataBasePath UTF8String], &contactDB) == SQLITE_OK) {
+        NSString *sql_makeJOSN = [NSString stringWithFormat:@"SELECT * FROM GPSTable"];
+        sqlite3_stmt *statement;
+        
+        if (sqlite3_prepare_v2(contactDB, [sql_makeJOSN UTF8String], -1, &statement, NULL)==SQLITE_OK) {
+            if (sqlite3_step(statement) == SQLITE_ROW) {
+                
+                while (sqlite3_step(statement) == SQLITE_ROW) {
+                    
+                    //NSLog(@"{timestamp: '%s', source: \"iOS\", gps: {latitude: %s, longitude: %s, altitude: %s, timerInterval %s}, region: {\"identifier\": \"%s\"}}", sqlite3_column_text(statement, 4),  sqlite3_column_text(statement, 1), sqlite3_column_text(statement, 2), sqlite3_column_text(statement, 3), sqlite3_column_text(statement, 4), sqlite3_column_text(statement, 5), sqlite3_column_text(statement, 6));
+                    
+                    
+                    JSONString = [NSString stringWithFormat:@"{timestamp: '%s', source: \"iOS\", gps: {latitude: %s, longitude: %s, altitude: %s, timerInterval %s}, region: {\"identifier\": \"%s\"}},", sqlite3_column_text(statement, 4),  sqlite3_column_text(statement, 1), sqlite3_column_text(statement, 2), sqlite3_column_text(statement, 3), sqlite3_column_text(statement, 4), sqlite3_column_text(statement, 5), sqlite3_column_text(statement, 6)];
+                    [JSONArray addObject:JSONString];
+                    
+                }
+                sqlite3_finalize(statement);
+                sqlite3_close(contactDB);
+                [self postJSON];
+                
+            }
+        }
+    }
+}
+
+-(void)postJSON
+{
+    
+    NSData *jsonData;
+    NSString*jsonString;
+    
+    jsonData = [NSJSONSerialization dataWithJSONObject:JSONArray options:0 error:nil];
+    jsonString = [[NSString alloc]initWithData:jsonData encoding:NSUTF8StringEncoding];
+    
+    NSString *postLength = [NSString stringWithFormat:@"%lu", (unsigned long)[jsonData length]];
+    
+    
+    // Be sure to properly escape your url string.
+    //string for the URL request
+    NSString *myUrlString = @"https://cpdev.dockrzone.com/api/v1/polls/vote/";
```

Comment:
as in the android version, the URL should be passed by parameter on starting the plugin, not hard coded.